app/src/androidTest/java/org/mozilla/fenix/ui/robots/HomeScreenRobot.kt

```diff
@@ -82,6 +85,18 @@ class HomeScreenRobot {
             return ThreeDotMenuRobot.Transition()
         }
 
+        fun openSearch(interact: SearchRobot.() -> Unit): SearchRobot.Transition {
+            mDevice.wait(Until.findObject(By.text("Search or enter address")), waitingTime)
+            navigationToolbar().perform(click())
+
+            SearchRobot().interact()
+            return SearchRobot.Transition()
+        }
+
+        fun dismissOnboarding() {
```

Comment:
This is a much simpler way to dismiss onboarding. Do you think this is sufficient to cover https://github.com/mozilla-mobile/fenix/issues/3777 or do we still want a flag?

Reply:
#3777 would still be a preferable way to disable onboarding screen, because it eliminates unnecessary steps for each test. But for now, this looks good

Reply:
👍